lib/ansiblelint/runner.py

```diff
@@ -1,23 +1,33 @@
 """Runner implementation."""
 import logging
 import os
-from typing import List, Set
+from typing import TYPE_CHECKING, Any, FrozenSet, List, Set
 
 import ansiblelint.file_utils
 import ansiblelint.skip_utils
 import ansiblelint.utils
+from ansiblelint.errors import MatchError
+from ansiblelint.rules.LoadingFailureRule import LoadingFailureRule
+
+if TYPE_CHECKING:
+    from ansiblelint.rules import RulesCollection
 
-from .errors import MatchError
-from .rules.LoadingFailureRule import LoadingFailureRule
 
 _logger = logging.getLogger(__name__)
 
 
 class Runner(object):
     """Runner class performs the linting process."""
 
-    def __init__(self, rules, playbook, tags, skip_list, exclude_paths,
-                 verbosity=0, checked_files=None) -> None:
+    def __init__(
+            self,
+            rules: "RulesCollection",
```

Comment:
I'm pretty sure that stringifying the type is not necessary, is it?

Reply:
It is, pylint would complain if not. Is not the mypy itself but pylint needs it.